src/reducers/paths.reducer.js

```diff
@@ -42,12 +53,14 @@ export default (state: State = initialState, action: Action) => {
 //
 //
 // Helpers
-export const getDefaultParentPath = () =>
+export const getDefaultParentPath = () => {
   // Noticing some weird quirks when I try to use a dev project on the compiled
   // "production" app, so separating their home paths should help.
-  process.env.NODE_ENV === 'development'
+  const { paths = {} } = getInitialState();
```

Comment:
If this needs access to the state, it should become a selector instead of a helper.

I have an idea for how to clean this up a bit. Let's use the `getDefaultParentPath` helper logic for the initial value for the state:

```js
const initialState = {
  defaultHome: process.env.NODE_ENV === 'development'
    ? `${os.homedir()}/guppy-projects-dev`
    : `${os.homedir()}/guppy-projects`,
  byId: {},
}
```

Then, the selector can just return that value:

```js
export const getDefaultProjectHomePath = (state: State) => {
  return state.paths.defaultHome;
}
```

You can use this in `mapStateToProps` by passing it the state.

Reply:
Need to be careful about the migration tho, it will be `undefined` for existing users